style.css

```
@@ -0,0 +1,200 @@
+@font-face {
+font-family: "Pretendard-Regular";
+src: url('https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
+font-weight: 400;
+font-style: normal;
+}
+
+*{
+    box-sizing: border-box;
+    font-family: Pretended;
+}
+
+body{
+    margin: 0;
+}
+
+.header-container{
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 10px 200px 9px 200px;
+}
+
+.logo{
+    width: 40px;
+    margin-right: 10px;
+}
+
+.logo-box{
+    display: flex;
+    align-items: center;
+}
+
+.title{
+    font-family: "ROKAF Sans";
```

font-family를 지정할 때는 혹시 없을 경우를 대비해서 sans-serif 체를 마지막에 주로 넣어서 작업하고 있습니다. 
```suggestion
    font-family: "ROKAF Sans" , sans-serif;
```